charts/amazon-cloudwatch-observability/values.yaml

```diff
@@ -602,6 +608,37 @@ agent:
         }
       }
     }
+  windowsDefaultConfig:
```

Comment:
This content actually now looks identical to the linux default above.
It might actually be cleaner to now have `cloudwatch-agent-windows` in the `agents` list above and drop this windowsDefaultConfig field here. We can instead add an `os` field thats either `linux` or `windows` and spin up all other resources based on that flag.

So `agents` would look like:
```
agents:
- name: cloudwatch-agent
- name: cloudwatch-agent-windows
  os: windows
```

Reply:
Its fine if we do this in a separate PR.